### ISSUE TITLE
Governor starts in performace in Openelec

### ISSUE THREAD

**Issue by mobiusloop** [maxnet/berryboot#99]
Is there a parameter that can be inserted in the config file of Berryboot or Openelec to force the governor into "ondemand" instead of the current default "performance" mode?

Thank you.

**Comment by sal666** [comment]
Make sure you have 'force_turbo=0' in config.txt.
You can also enter this command while in the console:
    echo "ondemand" | /sys/devices/system/cpu/cpu0/cpufreq/scaling_governor

(more info here: http://megakemp.com/2013/02/26/adventures-in-overclocking-a-raspberry-pi/)

**Comment by mobiusloop** [comment]
Thanks for the information.

I had the force_turbo=0 embedded in the config.txt and checked that it was there but Openelec stayed in "performance".

The echo "ondemand" | /sys/devices/system/cpu/cpu0/cpufreq/scaling_governor in an ssh session resulted in a "Permission denied" and no speed change.

I tried   sh -c "echo 'ondemand' /> /sys/devices/system/cpu/cpu0/cpufreq/scaling_governor" and this did get Openelec into ondemand mode.

Unfortunately, I will have to start a ssh session from my laptop to the rpi everytime I use Openelc.  If I get ambitious, I could look at modifying the .img file to embedded the scaling_governor command.  I have tried a few scaling_governor commands in the config file but no happiness.

Thanks again.
